### PR TITLE
Add event for splash screen closing in DynamoRevit.

### PR DESCRIPTION
**Pull request by reddyashish**
### Purpose

This is to add a event that will shut down the Dynamo model and enable Dynamo icon in Revit. 

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] Snapshot of UI changes, if any.

### Reviewers
@QilongTang @mjkkirschner

### CODE REVIEW

**Review by QilongTang — APPROVED** [review]

**Comment by wangyangshi** [comment]
thank you, Aaron, for the fixing, will this come into effective after merging new DC?

**Comment by QilongTang** [comment]
> thank you, Aaron, for the fixing, will this come into effect after merging new DC?

Yes, once the new DynamoCore is out and integrated, the close button on Splash screen should work and re-enable Dynamo button.

**Review by wangyangshi — APPROVED** [review]

**Comment by reddyashish** [comment]
@wangyangshi @QilongTang I don't have merge access to this. Can any of you do it.

**Pull request merged**
